sdk/template/AzureTemplate/Source/Dummy.swift

```diff
@@ -27,4 +27,3 @@
 import Foundation
 
 // Does nothing
-
```

Comment:
Is there a reason for this to exist?

Reply:
It's for engineering system testing. There has to be at least one source file.